requests_samples/upload_leads_csv.py

```diff
@@ -0,0 +1,120 @@
+"""
+    Usage:
```

Comment:
What's the difference between files in `request_samples` and `hustle_public_api_client/samples`?

Reply:
From the README:

```
Here we represent two ways of accessing the public api. One is for basic scripts with the requests library. 
The other is through a generated client library for accessing the Hustle Public API
```

And then I go on to explain the two. Should I be more explicit about when one might be more appropriate for the other? i.e: `Requests is lightweight and better suited for quick one off scripts and basic jobs, while you may want the typing safety and robustness of an auto-generated client for more critical regular jobs that are more integrated with your internal systems.`

Reply:
Ah that makes sense.
I think you explained it well. Now that you point it out it seems obvious. Not worth revising I think.